lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -1225,7 +1227,8 @@ class ConvertAtenContiguousOp : public OpConversionPattern<AtenContiguousOp> {
       return failure();
 
     Type resultType = getTypeConverter()->convertType(op.getType());
-    rewriter.replaceOpWithNewOp<tensor::CastOp>(op, resultType, adaptor.getSelf());
+    rewriter.replaceOpWithNewOp<tensor::CastOp>(op, resultType,
+                                                adaptor.getSelf());
```

Comment:
Looks like there are a few of these formatting changes. Did you add them yourself or did the linter do this? Should be fine, just wondering where they are coming from.